plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -607,18 +607,14 @@ private function get_setting_for_feature( string $feature_id, array $feature, bo
 			$disabled = true;
 			$desc_tip = __( 'WooCommerce Admin has been disabled', 'woocommerce' );
 		} elseif ( 'new_navigation' === $feature_id ) {
-			$needs_update = version_compare( get_bloginfo( 'version' ), '5.6', '<' );
-			if ( $needs_update && current_user_can( 'update_core' ) && current_user_can( 'update_php' ) ) {
-				$update_text = sprintf(
-				// translators: 1: line break tag, 2: open link to WordPress update link, 3: close link tag.
-					__( '%1$s %2$sUpdate WordPress to enable the new navigation%3$s', 'woocommerce' ),
-					'<br/>',
-					'<a href="' . self_admin_url( 'update-core.php' ) . '" target="_blank">',
-					'</a>'
-				);
-				$description .= $update_text;
-				$disabled     = true;
-			}
+			$update_text = sprintf(
+				// translators: 1: line break tag.
+				__( '%1$s The development of this feature is currently on hold.', 'woocommerce' ),
+				'<br/>'
+			);
+			$description .= $update_text;
+			update_option( $this->feature_enable_option_name( $feature_id ),'no' );
```

Comment:
This will always update the option even if it was already disabled, maybe we want to check if it needs updating first?
Also this means that for anyone who previously had the navigation enabled will have it auto disabled upon navigating to the Features setting page, this seems a little intrusive.

As per the discussion in the [original issue](https://github.com/woocommerce/woocommerce/issues/35157#issuecomment-1285680235), I think the logic should change a bit.
We should only have it set to disabled if the feature was already disabled, and allow to the user to manually disable it if it was enabled.

Adding the note that Jarek mentioned in the link above would also be great!

Reply:
@louwie17 thank you for pointing this out. 

> We should only have it set to disabled if the feature was already disabled

I included this variant in this PR and will work on the follow-up (navigation enabled case) in a sequel.